src/context/input_reader.rs

```diff
@@ -153,6 +153,59 @@ impl InputReader<'_, '_> {
                 value.into()
             }
             Binding::None => false.into(),
+            Binding::AnyDigital => {
+                if self.ignored(Binding::AnyDigital) {
+                    return false.into();
+                }
+                let value = match *self.gamepad_device {
+                    GamepadDevice::Single(entity) => {
```

Comment:
With `GamepadDevice::Single` I would still expect for other sources to be checked.

Reply:
`GamepadDevice::Single` only limits the gamepad input to specific device. All other input sources still used.
You can take a look at the `local_multiplayer` example. It doesn't use `AnyDigital`, but that's how other bindings work and I think we need to make `AnyDigital` consistent.

Checking the gamepad input would require another solution, but in this case I'd just use `bevy_input` instead of spawning a context for each connected gamepad.